.circleci/config.yml

```diff
@@ -36,40 +15,28 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-          - v3-rbenv-dependencies-20230418-{{ checksum ".ruby-version" }}
-          - v3-dependencies-v3-{{ checksum "Gemfile.lock" }}
-
-      - run:
-          name: rbenv update
-          command: |
-            rbenv_update_env
-
-      - save_cache:
-          paths:
-            - ~/.rbenv
-          key: v3-rbenv-dependencies-20230418-{{ checksum ".ruby-version" }}
+          - bundle-{{ checksum "Gemfile.lock" }}
```

Comment:
```suggestion
          - v1-gemfile-lock-{{ checksum "Gemfile.lock" }}
```

Reply:
OK

Reply:
It's done. Now we can invalidate the cache by modifying `CACHE_VERSION` from the CircleCI UI.